CMakeLists.txt

```diff
@@ -268,6 +268,8 @@ if(NOT Launcher_FORCE_BUNDLED_LIBS)
     find_package(ghc_filesystem QUIET)
 endif()
 
+include(ECMQtDeclareLoggingCategory)
+
```

Comment:
on the [`launcher/CMakeLists.txt`](https://github.com/PrismLauncher/PrismLauncher/pull/684/files#diff-42f96cf84d54866c2821d33afa39c85433ce6e405a74daf75da5e8bda53508c5) file, you're testing for the `KDE_INSTALL_LOGGINGCATEGORIESDIR` variable. This variable, however, would seem to me to never be available if we don't include the `KDEInstallDirs` module, right?

Considering the [`KDEInstallDirs6`](https://api.kde.org/ecm/kde-module/KDEInstallDirs6.html) module isn't yet ready for usage in applications, we could just try to import the [`KDEInstallDirs5`](https://api.kde.org/ecm/kde-module/KDEInstallDirs5.html) module on the Qt5 Linux builds, and later change it to just `KDEInstallDirs` when the Qt6 module is ready and they link it to the appropriate version.

Reply:
We include `KDEInstallDirs` in the main CMake file for Linux and other non macOS UNIX targets.

Reply:
hmm, it will have the wrong name for the Qt6 version then, no? It will install the Qt6 stuff in the Qt5 path o.O

Reply:
If you are building in an environment where ECM did not have Qt 6 support, then it will indeed use Qt 5 paths. On newer ECM versions `KDEInstallDirs` is effectively an alias for `KDEInstallDirs${QT_MAJOR_VERSION}`. See https://invent.kde.org/frameworks/extra-cmake-modules/-/blob/master/kde-modules/KDEInstallDirs.cmake

Reply:
ah ok, the docs say it's just a wrapper around the Qt5 one, so i just went with that :P